src/herbpy/herbrobot.py

```diff
@@ -472,7 +478,6 @@ def SetStiffness(self, stiffness, manip=None):
     def Say(self, words, block=True):
         """Speak 'words' using talker action service or espeak locally in simulation"""
         if self.talker_simulated:
-            import subprocess
```

Comment:
Nit: Should we really move this to an import at the top? We can do a lot of things without `Say`.